debian/confluent-schema-registry.postinst

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+set -e
+
+case "$1" in
+    configure)
+	_user=cp-schema-registry
+	_group=confluent
+
+	(getent group $_group 2>&1 >/dev/null) || addgroup --quiet --system $_group
+        adduser --system --disabled-password --disabled-login --home /var/empty \
+                --no-create-home --quiet --force-badname --ingroup $_group $_user
+
+	_permwarn=
+	for dir in /var/log/confluent ; do
```

Comment:
same question here as in #682 about permissions on other directories/the jar files.

Reply:
same answer!